code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -283,6 +285,30 @@
 	..()
 	. = 1
 
+/datum/reagent/consumable/caffeine
+	name = "Caffeine"
+	id = "caffeine"
+	metabolization_rate = 0.2 * REAGENTS_METABOLISM
+	description = "Essence of coffee."
+	color = "#482000" // rgb: 72, 32, 0
+	nutriment_factor = 0
+	taste_description = "bitterness"
+	glass_icon_state = "glass_brown"
+	glass_name = "glass of coffee"
```

Comment:
Wouldn't this be the default 'glass of what' like normal compounds instead of pseudo coffee? Im not opposed to caffeine as it is but this seems a little copy-pasted. [Drinking a cup of caffiene over 20u (or least enough in the range of 50u) in real life would send you into cardiac arrest.](https://edition.cnn.com/2017/03/31/health/worlds-strongest-coffee/index.html)